tracer_helpers.go

```diff
@@ -0,0 +1,59 @@
+package lightstep
+
+import (
+	"golang.org/x/net/context"
+
+	opentracing "github.com/opentracing/opentracing-go"
+)
+
+// Flush forces a synchronous Flush.
+func Flush(ctx context.Context, tracer opentracing.Tracer) {
+	lsTracer, ok := tracer.(Tracer)
+	if !ok {
+		onEvent(newEventUnsupportedTracer(tracer))
+		return
+	}
+	lsTracer.Flush(ctx)
+}
+
+// CloseTracer synchronously flushes the tracer, then terminates it.
+func Close(ctx context.Context, tracer opentracing.Tracer) {
+	lsTracer, ok := tracer.(Tracer)
+	if !ok {
+		onEvent(newEventUnsupportedTracer(tracer))
+		return
+	}
+	lsTracer.Close(ctx)
+}
+
+// GetLightStepAccessToken returns the currently configured AccessToken.
+func GetLightStepAccessToken(tracer opentracing.Tracer) (string, error) {
+	lsTracer, ok := tracer.(Tracer)
+	if !ok {
+		return "", newEventUnsupportedTracer(tracer)
+	}
+
+	return lsTracer.Options().AccessToken, nil
+}
+
+// DEPRECATED: use Flush instead.
+func FlushLightStepTracer(lsTracer opentracing.Tracer) error {
```

Comment:
nit: call out to `Flush(...)` instead of copying the code.

Reply:
Can't if we want to preserve original behavior (returning a typecheck error).

Reply:
Ah I see.

One thought, adding an `error` to the return of `Flush(...)` would be backward compatible (because if you don't look that response, it's not going to hurt to return it.

That would mean that both methods both `emit` and `return` the same `error`. I get that might not be what you are going for.

👍